cmake/flash/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
-assert_not(FLASH_SCRIPT "FLASH_SCRIPT has been removed; use BOARD_FLASH_RUNNER")
-assert_not(DEBUG_SCRIPT "DEBUG_SCRIPT has been removed; use BOARD_DEBUG_RUNNER")
+# FIXME: This will be fixed once the scripts are written for xcc
```

Comment:
It seems unnecessary to put this here since the scripts are already written.  They're just added later in the series.

Why not rebase fbc49163a ("intel_s1000: scripts: debug, debugserver and flash scripts") to just before this commit, so that Zephyr's history never permits setting the deprecated variables?

Reply:
Done. Thanks for the suggestion.